ironic/tests/unit/drivers/modules/drac/test_raid.py

```diff
@@ -926,19 +1112,37 @@ def test_create_configuration_with_backing_physical_disks(
     @mock.patch.object(drac_common, 'get_drac_client', spec_set=True,
                        autospec=True)
     @mock.patch.object(drac_raid, 'list_physical_disks', autospec=True)
+    @mock.patch.object(drac_raid, 'change_physical_disk_state', spec_set=True,
```

Comment:
change_physical_disk_state is not a method in drac_raid, which is the same as ironic.drivers.modules.drac.raid.  As a result, the line above and below should be removed.